fpmsyncd/routesync.cpp

```diff
@@ -1714,6 +1738,12 @@ void RouteSync::onRouteMsg(int nlmsg_type, struct nl_object *obj, char *vrf)
         fvVector.push_back(wt);
     }
 
+    if (route_eligible_for_fallback_to_default_route)
+    {
+        FieldValueTuple tag("fallback_to_default_route", "true");
```

Comment:
Where is this field processed?

Reply:
@dgsudharsan : Here https://github.com/sonic-net/sonic-swss/pull/3389